src/postamble_minimal.js

```diff
@@ -123,6 +123,9 @@ function loadWasmModuleToWorkers() {
 /*** ASM_MODULE_EXPORTS_DECLARES ***/
 #endif
 
+/*** STATIC_DYNCALL_SIG_FUNCTIONS ***/
```

Comment:
Unrelated to this PR but other places where do late substitution I've been using `<<<   ...  >>>` to avoid confusion with jsifier's `{{{ ... }}}`.  Should we switch these to use the same pattern?

Seems like it would be good to distinguish comments from substitution patterns, and also will correctly error out of we forget/fail to make the substitution.

Reply:
Yeah maybe <<< >>> is good, I'll have that in a followup. Although now I wonder if I need this at all. I'll see after I get through the test suite.